apps/meteor/app/livechat/server/lib/Livechat.js

```diff
@@ -955,6 +957,7 @@ export const Livechat = {
 		for await (const room of cursor) {
 			await FileUpload.removeFilesByRoomId(room._id);
 			await Messages.removeByRoomId(room._id);
+			await ReadReceipts.removeByRoomId(room._id);
```

Comment:
Can you change this to an `allSettled`?

Reply:
should we rather call `deleteRoom`? 😬

Reply:
Not sure if we can call `deleteRoom`. This would erase all subscriptions, but looking at the next line, looks like that's not what we want :thinking: maybe @KevLehman can help us here :thinking: 
Changing this to an `allSettled` for now